packages/bento-design-system/src/createBentoComponents.ts

```diff
@@ -254,3 +283,6 @@ export function createBentoComponents<
     useComponentsShowcase,
   };
 }
+
+type R<SprinklesFn extends typeof bentoSprinkles, CustomChipColor extends string = never>
+   = ReturnType<typeof internalCreateBentoComponents<SprinklesFn, CustomChipColor>>;
```

Comment:
this is the new thing in 4.7, i.e. the fact we can do `typeof foo<A, B>` where previously that would have been a parser error.

In fact, `prettier` will also choke on it since the parser hasn't been updated to suppor that syntax yet (it's called "instantiation expressions", see https://devblogs.microsoft.com/typescript/announcing-typescript-4-7-beta/#instantiation-expressions)